pages/agent/index.tsx

```diff
@@ -0,0 +1,238 @@
+        <p>
+          Le bouton AgentConnect vous permet d’accéder facilement à l’espace
+          agent grâce à votre adresse email professionnelle.
+        </p>
+      </div>
+      <div className="fr-col-12  fr-col-md-4">
+        <h3>Encore plus de données</h3>
+        <img src="/images/lp-agent/data-classification 1.svg" alt="" />
+
+        <p>
+          En tant qu’agent public, accédez automatiquement aux informations
+          protégées des entreprises (non diffusibles, statuts, actes, bilans
+          financiers).
+        </p>
+      </div>
+      <div className="fr-col-12  fr-col-md-4">
+        <h3>Une équipe à votre écoute</h3>
+        <img src="/images/lp-agent/string-phone-communication 1.svg" alt="" />
+        <p>
+          Vous rêvez d’une nouvelle donnée, d’une fonctionnalité, vous
+          rencontrez un problème ? Vos retours sont très précieux pour continuer
+          d’améliorer cet espace.
+        </p>
+      </div>
+    </section>
+    <section>
+      <h2>Comment l’Annuaire vous aide au quotidien ?</h2>
+      <div
+        className="fr-grid-row fr-grid-row--gutters"
+        style={{ marginTop: '1rem' }}
+      >
+        <CaseExample
+          title="Vérifiez facilement l’éligibilité des entreprises en consultant leur bilan financier et leurs statuts."
+          description="Pour instruire vos demandes de subventions"
+          image="/images/lp-agent/woman-work-with-dashboard 1.svg"
+        />
+        <CaseExample
+          title="Accédez directement aux informations des non diffusibles, sans avoir à leur demander de documents en plus."
+          description="Pour aider les entreprises individuelles"
+          image="/images/lp-agent/Group.svg"
+        />
+      </div>
+    </section>
+    <Container
+      style={{
+        background: 'var(--background-alt-blue-france)',
+      }}
+    >
+      <section
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          gap: '1rem',
+        }}
+      >
+        <h2>Rejoignez les agents qui utilisent déjà l’espace agent !</h2>
+        <ButtonAgentConnect />
+      </section>
+    </Container>
+    <section>
+      <h3>
+        L’Annuaire des Entreprises est opéré par la DINUM, avec le partenariat
+        de l’INSEE, l’INPI, l’ADEME etc.
+      </h3>
+      <div className="logo-soup">
+        <img src="/images/logos/ademe.svg" alt="ADEME" />
+        <img src="/images/logos/insee.svg" alt="INSEE" />
+        <img src="/images/logos/inpi.svg" alt="INPI" />
+        <img src="/images/logos/agence-bio.svg" alt="Agence bio" />
+        <img src="/images/logos/ess-france.svg" alt="ESS-France" />
+        <img
+          src="/images/logos/marche-inclusion.svg"
+          alt="Marché de l'inclusion"
+        />
+        <img src="/images/logos/dila.svg" alt="DILA" />
+
+        <img src="/images/logos/vies.svg" alt="VIES" />
+      </div>
```

Comment:
You can dynamically loop through `administrationMetaData`
see https://github.com/etalab/annuaire-entreprises-site/blob/12c3c951a28de885c245d41abe015110cf916b6d/components/footer/index.tsx#L252